docs/how-tos/dbt_extensions.rst

```diff
@@ -1,14 +1,20 @@
 .. _dbt-extensions:
 
-DBT extensions
-**************
-
-To extend the DBT project, you can use the following Tutor variables:
-
-- **DBT_REPOSITORY**: A git repository URL to clone and use as the DBT project.
-- **DBT_BRANCH**: The branch to use when cloning the DBT project.
-- **DBT_PROJECT_DIR**: The directory to use as the DBT project.
-- **EXTRA_DBT_PACKAGES**: A list of python packages for the DBT project to install.
-- **DBT_ENABLE_OVERRIDE**: This variable determines whether the DBT project override feature
-  should be enabled or not. When enabled, it allows you to make changes to the **dbt_project.yml**
-  and **packages.yml** files using the tutor patches: `dbt-packages` and `dbt-project`.
+Extending dbt
+*************
+
+As noted in `dbt concepts`_, you can install your own custom DBT packages to apply your own transforms to the event data
+in Aspects.
+
+To change which DBT packages are installed, use the following Tutor variables:
+
+- **EXTRA_DBT_PACKAGES**: A list of pip dbt packages for Aspects to install. Add your custom ddt packages here.
```

Comment:
nit: ddt -> dbt

Reply:
I think the preferred way to customize dbt is to change the `DBT_REPOSITORY` / `DBT_BRANCH` / `DBT_REPOSITORY_PATH` to your dbt project, and have that project make our dbt package a requirement. Is that you how you did it @Ian2012 ?

Reply:
Yes, you can update it to point to the main branch (to be always up to date) but it's better to have a pinned version so we don't introduce breaking changes

Reply:
Oh interesting, I thought that `EXTRA_DBT_PACKAGES` was for adding custom dbt packages, rather than encouraging people to fork `aspects-dbt` or run with a different version?

Reply:
No, `EXTRA_DBT_PACKAGES` is a list of python requirement that will be installed before your custom version of `aspects-dbt` will be run.

While, you can change your DBT_REPOSITORY and create a little `packages.yml`  with `aspects-dbt` as a package. This will allow you to have all the base functionality of aspects while creating your custom models without forking. See this [commit](https://github.com/eduNEXT/dbt-aspects-unidigital/commit/1ff80003429d435cc73ea37421f3786b9658d7ee) for an example

Reply:
Oh cool, thank you for these details @Ian2012 ! I'll incorporate them here.

Reply:
How's this? https://docsopenedxorg--111.org.readthedocs.build/projects/openedx-aspects/en/111/how-tos/dbt_extensions.html

Reply:
Great, just missing two pieces I forgot to mention. You can create the project with [dbt init](https://docs.getdbt.com/reference/commands/init) and you must update the `dbt_project.yml` file to use the aspects [profile](https://github.com/eduNEXT/dbt-aspects-unidigital/commit/50e4c154555a67a1a98bc61cbe0533b4765446a9)

Reply:
Nice gotcha, thanks @Ian2012 !
This is looking better: https://docsopenedxorg--111.org.readthedocs.build/projects/openedx-aspects/en/111/how-tos/dbt_extensions.html